resources/dicts/relationship_events/normal_interactions/dislike/increase.json

```diff
@@ -207,7 +389,14 @@
 		"interactions": [
 			"m_c accuses r_c of being a bad leader.",
 			"m_c accuses r_c of not taking care of the Clan.",
-			"m_c accuses r_c of not putting the Clan first."
+			"m_c accuses r_c of not putting the Clan first.",
+			"m_c accused r_c of leading the Clan toward the Dark Forest.",
+            "m_c wonders what stupid decision r_c will make next that'll harm the Clan.",
+            "m_c said aloud how {PRONOUN/m_c/subject} couldn't wait for m_c to just die so someone better could become leader.",
```

Comment:
```suggestion
            "m_c said aloud how {PRONOUN/m_c/subject} couldn't wait for m_c to die so that someone better could become leader.",
```